Flutter User Facing API.dart

```diff
@@ -0,0 +1,71 @@
+class Manager {
+  Future<Null> createClient(String restoreStateIdentifier) async
```

Comment:
Do we need `Future<Null>`? it is said that `Future<void>` is a better idea in most cases.

Reply:
True